### DISCUSSION TITLE
Favorites new icons

### DISCUSSION THREAD

**Discussion by DIrgi** [devemux86/cruiser#507] [Q&A] [answered]
There are some pre-given symbols for pois available and we can select a color for them.
<img src="https://github.com/devemux86/cruiser/assets/57968526/abd96a28-19c2-49d8-a55b-f42bef52cad2" width="50%"/>
For my personal belongings I miss some. And some I'll never use and need. For sure I can use a cocktail symbol für a tunnel symbol (just an example) but well....it's just a bypass solution.

I was already searching for the given symbols (test for replacement) but I'm not very linux(android) familar and don't know where all data might spread (in which folders) and the symbols I found where not the shown ones so I assume they are stores in one data file as an xml description.

Question:
Can we use own symbols (for the future) and import them?
Afaik a symbol can/has to be a svg (vector) files or maybe even a png.

Cheers, Dirk

**Comment by devemux86** [comment]
@DIrgi thanks for the suggestion.

User icons from the storage is not easy, especially as Android has more storage restrictions.

We can always add more icons in the favorites from the open source [Material icons collection](https://fonts.google.com/icons?icon.set=Material+Icons&icon.style=Filled).
(we already did it in the past)

**Reply by DIrgi** [reply]
Hmmm...only "half way" satisfying but...does that mean I can reply for adding some more symbols out of the icon collection content?

**Reply by devemux86** [reply]
> does that mean I can reply for adding some more symbols out of the icon collection content?

That's right, we can choose anything from Material icons and add them in favorite icons selection screen.

**Reply by Ciutnik** [reply]
I have already learned to live with the set of icons we have to choose from, but I admit that I would like to see a few new ones to mark various attractions 🙂

**Reply by DIrgi** [reply]
Fine, based on the icons collection,
I would apply for:

(revised)


Device:
- Water
![image](https://github.com/devemux86/cruiser/assets/57968526/7d3f2fc7-8bc6-438b-a707-5fe97d8993b8)


Maps:

- Forrest
![image](https://github.com/devemux86/cruiser/assets/57968526/7e37175e-d410-4072-86d8-1ba0cb1dac3b)

- Castle
![image](https://github.com/devemux86/cruiser/assets/57968526/50d7edd5-b028-444b-839b-03c9ee7bf595)

- Warehouse
![image](https://github.com/devemux86/cruiser/assets/57968526/43977ebb-bc5d-44aa-a777-938991a93131)

-  Flight
![image](https://github.com/devemux86/cruiser/assets/57968526/625e4618-5c31-44d9-b062-fe85289b79d7)


Image:

- Panorama
![image](https://github.com/devemux86/cruiser/assets/57968526/bc84b136-4f99-4e77-99ad-2588af7bd80d)

- Looks
![image](https://github.com/devemux86/cruiser/assets/57968526/0589111c-974c-4bc0-93d1-23e165f0d0d1)

Places:
- Pool 
![image](https://github.com/devemux86/cruiser/assets/57968526/6a5b1acf-d1a7-4086-afdb-6e2d6b26f3b4)

- Beach access
![image](https://github.com/devemux86/cruiser/assets/57968526/422b60fe-278e-4832-98fd-c24cc6ed5ce1)

-  Cottage
![image](https://github.com/devemux86/cruiser/assets/57968526/b5f7ea6a-47a0-4812-afb5-93a6d58b0a78)

Social:

- Volcano
![image](https://github.com/devemux86/cruiser/assets/57968526/1e5ffdc9-52f2-4bd9-9c3c-0ac3843bae18)

**Reply by devemux86** [reply]
@DIrgi thanks, please select from [Material icons](https://fonts.google.com/icons?icon.set=Material+Icons&icon.style=Filled) (not symbols).

They are different, we only use the icons collection.

**Reply by DIrgi** [reply]
@Ciutnik check out the icon catalog, there are several icons for attractions.
the collection is somehow huge but in some cases still limited. i.e. I found in the whole catalogue no sign for a road tunnel, just a subway tunnel, so I decided to select the "folded hands" which look strange but somehow like a tunnel

**Reply by 0709wiwiwi** [reply]
An oft-repeated question but never solved.  I have a suggestion that does work. Select **ANY** image and show it as a thumbnail icon on the map at **ANY** waypoint location. Both by the Locus map and the gpx viewper app you can already (manually) demonstrate this. However, the practical finishish touch (automatic show the thumbnail icon) is not yet offered in either app. 
If you are a bit experimental, give this a try and you will find that it is not only simple but works well and, by the way, can be combined perfectly with the gpx navigation track. Some user interest may ensure a possible realisation.  To that end, read the information document attached here: [Image and icon transfers](https://github.com/devemux86/cruiser/discussions/127#discussion-5077760)

**Reply by devemux86** [reply]
@0709wiwiwi this is not so simple.

There are many image formats with best the SVG so we can adjust size, color, outline, etc.
That's why we only use the icons (and not the symbols) from the Material collection.

Also reading user images and storing them locally, transfer them, etc. is complicated.

As this feature already exists in Cruiser with icon collection that can be enriched by your suggestions,
this could be considered in the future after we complete other more important non-existing features.

**Reply by DIrgi** [reply]
Ok, I revised my selection based on Material Icons only.
Some are just placeholders for symbols they don't exist i.e. "Looks" as cave, cavern or tunnel

**Reply by 0709wiwiwi** [reply]
These (svg) are the in the app provided high quality (limited range) icons.  
Best with svg yes sure perfect quality indeed.  
But that is not what I do suggest.

_I mainly advertise to users who might be interested in this but had never heard of the proposal._

There are regularly requests in diverse gps apps for specific icons and images. 
By the proposal even transfer pdf files or other specific files. (Possible and tested)
Icons requests like a tunnel icon or specific icons as used for boats & ships mining etc.  
No gps app can ever offer the thousands of such diverse icons that people have in mind.
![image](https://github.com/devemux86/cruiser/assets/54167201/9123797b-e3c5-4e26-8a36-a0e577935f7b)

Note.
This was already discussed in the gpx working group in the far distant past. 
But apart from a lot of talking, almost nothing extra since Y2004 is being realised.  

Tunnel Icons.  Less quality indeed but who is going to hang around with his nose on the navigation screen to check its quality ? 
[Tunnel_icons.zip](https://github.com/devemux86/cruiser/files/15063465/Tunnel_icons.zip)

There are definitely better and more nice tunnel icon images out there.  But not everything is royalty-free.  
If the app developper does not offer anything wrong, no problem arises for the pro app either.

**Reply by DIrgi** [reply]
@0709wiwiwi Designing an icon by my own and save it as jpg, png or svg or what else for my personal use is not the problem.

**Reply by 0709wiwiwi** [reply]
> after we complete other more important non-existing features.

Completely agree Emux.  
Especially those features that more directly address the routing and navigation experiences and do optimise them even more. ;-).
I'm just trying to attract interest in this feature so that it doesn't get forgotten.  So If I notice that there are users here who may be very knowledgeable on that subject, I wake up ;-)

**Reply by Ciutnik** [reply]
I noticed a few more tourist attractions:

<img src="https://github.com/devemux86/cruiser/assets/122647782/e5c74fd1-1db8-406f-a498-eef322139563" width="50%"/>
<img src="https://github.com/devemux86/cruiser/assets/122647782/6965dea9-bef1-4579-85aa-3577b42925a7" width="50%"/>
<img src="https://github.com/devemux86/cruiser/assets/122647782/c2a2de90-119d-4d05-967b-ecb7cce3565d" width="50%"/>
<img src="https://github.com/devemux86/cruiser/assets/122647782/82926304-55e5-4509-8ebf-ec7491170d58" width="50%"/>

I didn't find anything on "monument/memorial/sculpture". But even without the above I can live. If you decide that they are unnecessary or that it is too much, nothing bad will happen if you give them up 🙂

Edit: Given our preference for winding roads, this seems particularly interesting  😁

<img src="https://github.com/devemux86/cruiser/assets/122647782/56673270-f488-4db5-9892-0df04a358f4e" width="50%"/>

**Reply by bios4** [reply]
I'll throw the following into the ring:

- "Attraction"
- "Commute"
- "Directions Boat" (could represent ferries)
- "Fastfood" (or "Lunch Dining")
- "Garage"
- "Icecream"
- "Local Pizza"
- "Local Taxi"
- "Nature"
- "Park"
- "Restaurant Menu" (or "Local Dining") (replacing the existing "Restaurant" icon)
- "Shower" (or "Bathroom")
- "Sports Bar" (suits better than the cocktail glass 😁)
- "Tire Repair"
- "Water Drop" (could represent drinking water points)

Strangely enough, there are no icons for campsites or tents...

**Reply by DIrgi** [reply]
@bios4 yes exactly - on one hand a lot of strange stuff but some important things like campground, (camping)trailer or mobile homes or a tunnel symbol - They are completely missing

**Reply by Ciutnik** [reply]
I found "monument/memorial/sculpture":

<img src="https://github.com/devemux86/cruiser/assets/122647782/c1bee496-dcc4-4f73-8f04-c68194023804" width="50%"/>

---

I admit that "Sports Bar" looks promising 🙂

<img src="https://github.com/devemux86/cruiser/assets/122647782/0dd66287-ff8b-4d51-b35e-a8b81fd71184" width="50%"/>

**Comment by devemux86** [comment]
Can someone summarize in one post what are all the new icon suggestions?

@Ciutnik you did a nice work last time, perhaps if you have some free time?

They must be from [Material icons](https://fonts.google.com/icons?icon.set=Material+Icons&icon.style=Filled), better in groups of 4, so the icons screen is symmetrical.
(unless we change its structure)

**Reply by DIrgi** [reply]
I can do it for my ones. As you know I‘m limited on the icon gallery, I try to find some Icons which are useful for pois I have in mind. Sometimes they are just close enough because there are no exact icon existing. I.e. the two bows or arrows was for me the closest icon (not by name) to describe a tunnel.

**Reply by devemux86** [reply]
@DIrgi thanks!

If you have free time, can you also check the posts above by @Ciutnik and @bios4 
and summarize the icons from all users in one list?

This would make my work easier and could be included much faster. :slightly_smiling_face:

**Reply by DIrgi** [reply]
I can do so probably that evening (we are not in a hurry since this is a nice to have feature but not mandatory). You want to see them all together as a icon overview tiles wall? Ok - But I don’t know their needs behind why they select these icons.
maybe we can put them together in a better sortiment with the existing ones to do not have symbols which look similar or dealing witch on topic (I.e. leisure or trafic), but spread over the whole assortment.

**Reply by devemux86** [reply]
> You want to see them all together as a icon overview tiles wall?

No, just a summary of all icon suggestions from all users in 1 list.
Also double checking that they all are [Material icons](https://fonts.google.com/icons?icon.set=Material+Icons&icon.style=Filled) (not symbols).

So I don't need to check all posts, collect them, check for duplicates, etc.
If someone can help with this, it would be nice. :slightly_smiling_face:

**Reply by DIrgi** [reply]
Ok I do so

**Reply by Ciutnik** [reply]
It's hard work to choose, for example, 12 icons to please everyone. Since 3 people expressed their willingness to introduce new ones, let each of them democratically propose 4? 

Since true democracy probably does not exist, I can give one icon each to the mentioned colleagues, while proposing only two myself 🙂:
- [Fort](https://fonts.google.com/icons?selected=Material%20Icons%3Afort%3A) or [Castle](https://fonts.google.com/icons?selected=Material%20Icons%3Acastle%3A)
- [Hail](https://fonts.google.com/icons?selected=Material%20Icons%3Ahail%3A) 

Let each of the remaining two originators choose 5 of their icons 🙂 What do you say @DIrgi @bios4 ?

**Reply by bios4** [reply]
I did not see any limitation for new-to-be-added icons, so why not just add ALL of the ones mentioned?

**Reply by DIrgi** [reply]
For sure, a castle or Fort can stay in general for a place/building to visit (btw. you forgot the mosque symbol 🙂 )
We can start with this to have a good outcome. And if there are some more needed in the future, we can add anyway.
I see that you even can develop icons  for the google pool and forward it to the them but to be honest I have not the time and knowledge to dive into it because I see as well some symbols missing what bios already said - i.e. campground... really very strange - one of the most useful symbols on maps. But it seems it's all about what a icon developer is interested in.

**Reply by DIrgi** [reply]
@Ciutnik @bios4 
Let me start collecting them and then we'll see the outcome - ok?

**Reply by devemux86** [reply]
@bios4 no limitation, just the total number of icons better be a multiple of 4,
as the icons list displays 4 icons per row, so we have a nice symmetry.

Although we can always change the number of icons per row in their selection screen.

**Reply by devemux86** [reply]
Note that the Material icons collection gets new icons all the time.
So it's worth revisiting from time to time.

**Reply by Ciutnik** [reply]
Don't forget that each icon can have many different colors! So far, I have distinguished between castles and ruins, observation towers and viewpoints, or... living and dead friends.

**Reply by bios4** [reply]
> Let me start collecting them and then we'll see the outcome - ok?

Alright, let's wait for the overview and decide then about the final list... 👍

There will be for sure some similar ones (e.g. "Castle" vs. "Fort", or "Cottage" vs. "Cabin") where we should decide to use ONE.
When we agreed on a joint list we finally check to have a multiple of 4 icons - and add up to a multiple of 4 if required.

**Comment by DIrgi** [comment]
### **Icon overview**

from material database
please find the icon proposals as an overview as discussed.
I added them all. 
Where you see ----------deleted
this can be skipped since there are similar one. Never the less if you need some for "fill up" you can take them.
Theres only one icon in the existing cruiser gallery which can be replaced (see remark)


### **Category:**

1.Action                     (2)
- Visability
- Commute

2.Device                     (1)
- Water                     

3.Maps                       (8)
- Forest                      
- Castle                                       --------deleted
- Warehouse                               --------deleted
- Flight
- Festival                     
- Fort                                           --------deleted
- Attractions                                --------deleted
- Directions Boat
- Fastfood
- Icecream
- Local Pizza                                ---------deleted
- Local Taxi                                  ---------deleted                
- Park                         
- Restaurant Menu                      ---------deleted
- Tire Repair                                ---------deleted
- Hail                          

4.Image                     (9)
- Panorama horizontal
- Looks
- Looks 1-6  (here we have the advantage to have numbers from 1 to 6 as a icon) - Good for track grade tagging
- Nature                     

5.Places                     (2)
- Pool                         
- Beach access
- Cottage                  (replace existing cabin sign with cottage sign)

6.Sozial                       (2)
 - Volcano
 - Health and Safety                  -----------deleted
 - Water Drop

7.Content                   (1)
- Gesture

8.Search                      (2)
-Garage                                    ----------- deleted
-Shower



I count 27 icons
36 icons can be on one cruiser symbol page.
Either you leave them blank or fill up with the "deleted" ones.

**Reply by devemux86** [reply]
@DIrgi thank you very much for your help and great work! :+1: 

@bios4 @Ciutnik and everyone else who suggested new icons, what do you think?

**Reply by bios4** [reply]
Im fine with the suggestions...

Regarding the icons with doubled (or similar) meaning, I suggest:
- "Water" (instead of "Pool"), because it could be used to indicate a lake (not necessary for "swimming")
- "Forest" (instead of "Nature")
- "Castle" (instead of "Festival"), but keep also "Fort" (e.g. to mark ruins)
- no real opinion regarding "Local Taxi" vs. "Hail"

**Reply by Ciutnik** [reply]
Although I really didn't expect that SO MANY new icons would be needed, I respect it and understand it - I also like having a choice 🙂
I fully agree with @bios4 about the similar choices described above, including that I would also like "Fort" (I love them!).

 As for "Local Taxi" vs. "Hail", I'm definitely in favor of "Hail" (as e.g. "monument") because we already have a "Directions car" icon!
The difference is really not worth duplicating the icon. Look:

<img src="https://github.com/devemux86/cruiser/assets/122647782/284c3ab4-9c6b-4e17-b688-c883f47a65d3" width="50%"/>

<img src="https://github.com/devemux86/cruiser/assets/122647782/7f3675d4-7f1b-47b3-b004-0db82f096811" width="50%"/>

**Reply by devemux86** [reply]
Indeed please also compare with the existing icons, so we add new icons that are really different.

**Reply by devemux86** [reply]
Have you all decided which final icons we can include?
Which are different from each other and from the existing ones.

Can someone please make the final list? @DIrgi ?

**Reply by devemux86** [reply]
> can it be possible to see at least the name (permanently by setting activation) in the map next to the POI? - It can be then even smaller.

Probably as an option, as it could fill the map with texts.

@DIrgi Please create a new topic with the suggestion so it doesn't get forgotten here.

About possible extra fields for favorites (and waypoints) we have some discussions:
- https://github.com/devemux86/cruiser/discussions/346
- https://github.com/devemux86/cruiser/discussions/401

**Comment by DIrgi** [comment]
So I have recently updated the list. Now you can take it. Further comments you'll see there.

**Reply by bios4** [reply]
What does that "deleted' mean???
Lots of them should definitely be NOT deleted...

I am confused.

**Reply by DIrgi** [reply]
I had the "instruction" to opt out "similar lookings". (rememeber there where only two in the beginning which where "out").
If you want to have them in, feel free to go through all of them compare and forward your "own" list.
Have fun....

**Reply by DIrgi** [reply]
It’s nothing deleted so far, just if necessary

**Reply by bios4** [reply]
My comment was not meant to be offensive...

> I had the "instruction" to opt out "similar lookings". (rememeber there where only two in the beginning which where "out").

I know - I am just surprised as it seems you "deleted" exactly the ones @Ciutnik and myself proposed to be used instead of their "twins".

**Reply by bios4** [reply]
> feel free to go through all of them compare and forward your "own" list

Alphabetically ordered:
- Attractions --------delete
- Beach access
- Castle
- Commute
- Cottage (could replace the existing "Cabin" icon)
- Directions Boat
- Fastfood
- Festival --------delete
- Flight
- Forest
- Fort (could represent ruins)
- Garage
- Gesture (I think this could be interpreted as "winding roads"; could potentially replace the existing icon which is shown directly below the red motorcycle in the image of the first post [didn't find its name])
- Hail
- Health and Safety -----------delete (is already available in existing icons as "Local Hospital")
- Icecream
- Local Pizza ---------delete
- Local Taxi ---------delete (suggestion was to use "Hail" instead)
- Looks ---------what could this represent?
- Looks 1-6 (could be used for track grade tagging)
- Nature ---------delete (use "Forest" instead)
- Panorama Horizontal (or "Vrpano" as alternative)
- Park ---------delete (use "Forest" instead)
- Pool ---------delete (use "Water" instead)
- Restaurant Menu ---------delete (is already available in existing icons as "Restaurant")
- Shower
- Tire Repair
- Visibility
- Volcano
- Warehouse --------delete
- Water
- Water Drop

**Reply by Ciutnik** [reply]
> Looks ---------what could this represent?

It looks like a **tunnel** 🙂 
I guess that was the idea.

**Reply by DIrgi** [reply]
You see., you recognize it so it's a replacement symbol for a tunnel since a tunnel does not exist.

**Comment by devemux86** [comment]
Note that when we introduce favorite icons, it is difficult to remove or replace them later.

Users may already use them in their favorites. If we remove or replace them,
they will see the default icon, which is not nice.

So we better choose carefully what new icons we really want to add.

**Reply by DIrgi** [reply]
Nothing is deleted or removed so far and there’s just one proposal for „replacement „ but for sure, it can remain like it is to fulfill the boundary arguments.

**Comment by devemux86** [comment]
Analyzing your suggestions:

- forest and nature: there is park?
- garage and taxi: similar to existing car
- health_and_safety: there is existing local_hospital
- panorama: there is existing terrain?
- pool: similar to water?
- restaurant_menu: similar to existing restaurant
- volcano: is it needed?

The remaining 28 icons:
- attractions
- beach_access
- castle
- commute
- directions_boat
- fastfood
- festival
- flight
- fort
- gesture
- hail
- icecream
- local_pizza
- looks
- looks 1-6
- park
- shower
- tire_repair
- visibility
- sports_bar
- warehouse
- water
- water_drop

**Reply by DIrgi** [reply]
- panorama is i.e. for scenic view
- volcano: Idea behind have another (further) mountain, peak, saddle symbol since all other are similar to this:
![image](https://github.com/devemux86/cruiser/assets/57968526/cf7d3069-b73a-49bf-a863-48d945b65498)
 to i.e. differentiate between saddle or peak or even a worthwhile mountain pass.

- pool vs water: (similar? yes/no) the pool symbol is a symbol for a pool or a lake/sea area where you can swim. Water can be something different (not necessary/possible for swimming)

**Reply by Ciutnik** [reply]
> Water can be something different (not necessary/possible for swimming)

I suggest you mark the water for swimming in green, the water forbidden to swim in red, and the unknown in blue 🙂

**Comment by devemux86** [comment]
28 new favorite icons (total 64):

<img src="https://github.com/devemux86/cruiser/assets/3484020/86769317-6916-42ea-a9fb-13baa0a95b32" width="50%"/>

**Reply by devemux86** [reply]
The new favorite icons are in the latest:
- https://github.com/devemux86/cruiser/discussions/482

We can see if more icons are needed or changes to new icons
or a different order of the icons in the selection screen.

**Comment by devemux86** [comment]
4 more favorite icons (total 68) in the latest:
- https://github.com/devemux86/cruiser/discussions/482

vrpano, pool, volcano and videocam.
Increased spacing on selection screen with different layout in landscape.

And use the looks 1-6 outline icons. Are they better than the filled icons?

<img src="https://github.com/devemux86/cruiser/assets/3484020/ac234342-e6de-4e72-93a3-37a0ee1a5cf3" width="50%"/>

**Reply by bios4** [reply]
> And use the looks 1-6 outline icons. Are they better than the filled icons?

I liked the first ones (non-outlined) more to be honest!
I think they are better readable...

**Reply by DIrgi** [reply]
Me too, I think.
I just see the symbols in the mail but I think as bios4 says the other are better readable

**Reply by devemux86** [reply]
We can change the looks 1-6 again.
Even better remove the rectangle and only use the numbers with larger size to make them easier to read.

**Reply by devemux86** [reply]
<img src="https://github.com/devemux86/cruiser/assets/3484020/53d470bd-9170-435d-8024-361a029459ab" width="50%"/>

**Reply by bios4** [reply]
Hmm, ok... I still liked the initial "Looks 1-6" the most. 😀

What we have completely ignored so far are banks/cash machines/ATMs.

Yes, I know we have this "account_balance" icon in the list - but let's be honest: does it even remotely look like a bank? It looks more like an ancient building to me - like the Acropolis or the Austrian Parliament Building in Vienna...

Maybe you can add the "euro_symbol" (Cruiser is a european app, isn't it 😀) or alternatively "attach_money", "monetization_on", "atm" or "currency_exchange" to the list!?

**Reply by devemux86** [reply]
> I know we have this "account_balance" icon in the list - but let's be honest: does it even remotely look like a bank? It looks more like an ancient building to me - like the Acropolis or the Austrian Parliament Building in Vienna...

:smile: 

> Cruiser is a european app, isn't it

Not really, there are many users west of the Atlantic, Korea, etc.
Unlike apps that advertise "made in" and focus on their country, we are open to all users. :slightly_smiling_face:

**Reply by bios4** [reply]
>> Cruiser is a european app, isn't it

> Not really

But you are from Greece, aren't you?
And you develop Cruiser on your own (at least until now)...

**Reply by devemux86** [reply]
The favorite icons are not related to the region of development.
But like the app interface, languages, etc. should be usable by all users.

**Reply by devemux86** [reply]
I had looked at finance icons in the past.

Many of them have the dollar in variations. Searching "currency" there are also several others.
Could add "atm", although don't know if it's very readable. There is also "money".

**Reply by bios4** [reply]
> Could add "atm", although don't know if it's very readable. There is also "money".

The "money" icon is also not recognizable as a banknote...
My suggestions remain unchanged.

**Reply by devemux86** [reply]
Could we replace any of the new icons if one is not really needed?
Candidates could be "attach_money", "euro", ...

**Reply by bios4** [reply]
> Could we replace any of the new icons if one is not really needed?

You could add "attach_money" or "euro" or ... and replace either "festival" or "local_pizza" with it.
Plus, I would replace "warehouse" with "garage", as it is more related to vehicles/motorcycles.

**Reply by devemux86** [reply]
"garage" has a car icon, there is already another car.

**Reply by bios4** [reply]
Then please do as you wish.

**Reply by DIrgi** [reply]
When it comes to the looks 1-6, I suggest shown picture 1 or 3. These are the best ones to see the numbers.
The  outline icons are the "worst" one when it comes to visibilty. (My opinion)

**Comment by 0709wiwiwi** [comment]
Perhaps a strange question. 
What exactly do you want to do in the end with these personal favorites (gpx wpt) and their icons? 
Do you want to share YOUR personal favourites (wpt in gpx) with others, in addition to the route?

**Reply by DIrgi** [reply]
Me? - if you mean me - honestly I had just 4 or 5 in consideration I asked to add for.
No I had no plans to share but I'd like to have several different POIs, describing things which are for me important i.e. map conditions, track grades, quality, permissions.......
and I like to have POIs which are at least close by what they should stay for. I rather understand this
![image](https://github.com/devemux86/cruiser/assets/57968526/a485e484-0396-4a09-a864-3b1b03697169)
as as tunnel symbol than that:
![image](https://github.com/devemux86/cruiser/assets/57968526/c433dd6f-830a-4edd-ba87-1a12176c817d)

Me in person, 
I don't need Pizza, hiking, cocktail, electric fuel signs etc. but maybe others do (fair enough, I don't care about).

**Reply by 0709wiwiwi** [reply]
So more clearly. Next is what I expect ! Sharing between apps !
I do not expect to find an abundance of icons on the map, (No nightmare)
I expect much more icon fine functionality in navigation apps like Cruiser.
Be able to select and create **a few** individual wpt locations inclusive  icon.

So that also a gpx wpt with Garmin sym is shown with a best matching icon
I explicitly mention Garmin as this older icon set is the most popular range.
Even with a limited Garmin sym range offer by some matching icons is useful.

Find a simple gpz navtrack file inclusive a single gpx waypoint in attachment. (compressed gpx)
[Pause_Bar.gpz.txt](https://github.com/devemux86/cruiser/files/15146079/Pause_Bar.gpz.txt)

Locus shows the informative (Garmin) icons. (Locus supports 266 Garmin icons)
Gpx viewer shows  by own custom icon set.( Limited Garmin sym and icon range support).
Cruiser shows the **default** icon. (Cruiser anyway has sure some useful matching icons).

Apps cannot provide all (thousands) of icons. This demo gpz file contains the icon image.
Apps without internal best match icon can display the attached icon image. (**NOT YET  automatically**)

The panorama image.
Image 1 Cruiser. Display default icon. Tap point displays the description field.
Image 2 Locus. Display description field and more details of the gpx wpt.
Image 3 Locus. Display by internal Garmin gpx sym icon support. (Classic gpx)
Image 4 Locus. Display (demo) thumbnail by the gpz icon image transfer. (New by gpz)

![Pause_Bar display](https://github.com/devemux86/cruiser/assets/54167201/55297a7c-08be-4522-b6e5-ac93e8c563e2)
Garmin icon images on white background.
[Garmin_symbols_PNG_White_BG.zip](https://github.com/devemux86/cruiser/files/15146032/Garmin_symbols_PNG_White_BG.zip)

**Comment by devemux86** [comment, marked as answer]
Favorites many new icons are available in [Cruiser 4.0.0](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Comment by 0709wiwiwi** [comment]
Question. 
Cruiser shows a graphical list display of the current supported internal icons.
Is this information also available as a simple icon sym text list ? Or xxxx.svg  ?

**Reply by devemux86** [reply]
The favorite svg icons come from the [Material Icons collection](https://fonts.google.com/icons?icon.set=Material+Icons&icon.style=Filled).

Here is the current list: [icons.txt](https://github.com/user-attachments/files/19573109/icons.txt)
